VikingQuiz.UI/src/Components/ForgotPasswordComponent/ResetPasswordComponent/ResetPasswordComponent.tsx

```diff
@@ -0,0 +1,98 @@
+import * as React from 'react';
+import { Redirect } from 'react-router-dom';
+import InputData from 'src/entities/InputData';
+import HomeButton from 'src/Components/Buttons/HomeButton/HomeButton';
+import FormComponent from 'src/Components/FormComponent/FormComponent';
+import './ResetPasswordComponent.css';
+import { changePasswordValidator } from 'src/entities/Validation/validators';
+import { changeRule } from 'src/entities/Validation/rules';
+import HttpService from 'src/services/HttpService';
+import StorageService from 'src/services/StorageService';
+
+class ResetPasswordComponent extends React.Component<any, any>
+{
+    private readonly apiAddress: string = '/email';
+    private httpService: HttpService = new HttpService();
+    private storageService: StorageService = new StorageService();
+
+    constructor(props: any) {
+        super(props);
+
+        this.state = {
+            redirect: false,
+            showStatusMessage: false,
+            statusClass: '',
+            statusMessage: ''
+        };
+
+        const resetToken: string = this.props.match.params.token;
+        this.storageService.saveItem('token', resetToken);
+    }
+
+    public render() {
+        if (this.state.redirect) {
+            return (<Redirect to='/login' />);
+        }
+        return (
+            <div className='my-container'>
+                <HomeButton />
+                <div className='yellow-bg-message'>update password</div>
```

Comment:
Unless specified otherwise in the design, all displayed text must be properly written. Capital first letter in this case.